packages/modular-scripts/src/cli.ts

```diff
@@ -353,6 +372,81 @@ async function buildSequential(
   }
 }
 
+// currently assumes cypress/e2e tests are not set up at all.
+async function initE2E(someArgs: string) {
```

Comment:
Things that would make this "setup" useful is quite foggy at the minute.
It might be useful to have a modular type 'package' in the case things are added to the monorepo that do not have types i.e. are simply dragged in.